pkg/config/dp-server/config.go

```diff
@@ -37,6 +37,9 @@ type DpServerAuthConfig struct {
 	// Type of authentication. Available values: "serviceAccountToken", "dpToken", "none".
 	// If empty, autoconfigured based on the environment - "serviceAccountToken" on Kubernetes, "dpToken" on Universal.
 	Type string `yaml:"type" envconfig:"kuma_dp_server_auth_type"`
+	// UseTokenPath define if should use config for ads with path to token that can be reloaded.
+	// Default value: false
```

Comment:
We don't put default value in the comment. It's clear from the kuma-cp default reference